chaos_kitten/validators/profile_validator.py

```
@@ -0,0 +1,143 @@
+        severity = content.get('severity')
+        if severity not in self.VALID_SEVERITIES:
+            report.is_valid = False
+            report.errors.append(f"Invalid severity '{severity}'. Must be one of: {', '.join(self.VALID_SEVERITIES)}")
+            
+        # Validate category
+        category = content.get('category')
+        if category not in self.VALID_CATEGORIES:
+            # Check if it's a known category or valid string
+            if not isinstance(category, str):
+                 report.is_valid = False
+                 report.errors.append(f"Category must be a string")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)